### DISCUSSION TITLE
Hi everyone!

### DISCUSSION THREAD

**Discussion by caos21** [OpenMined/courses#145] [Introduce Yourself]
My name is Ben Santos writing from Canada. I worked on projects from different fields: Data Science, Software Security, Civil Engineering, and Computational Physics. Although these fields seem unrelated, I use and extend Free/Libre Open Source Software most of the time.

Generally speaking, I am very interested in privacy, transparency, and information flows. Specifically, right now, I want to dig into running models using encrypted data.

Thanks to the team for all your efforts and for giving these courses for free for the benefit of all.

**Comment by yemikifouly** [comment]
Hello Ben! 👋 Well, given those interests, you are definitely at the right place! 😃 Did you start the course? Are you enjoying it?
